src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/ComController.kt

```diff
@@ -100,4 +101,43 @@ class ComController(private val comService: ComService, private val notifyServic
   ) {
     notifyService.sendInitialLicenceCreateEmails(body)
   }
+
+  @PostMapping(
+    value = ["/case-search"],
+    produces = [MediaType.APPLICATION_JSON_VALUE]
+  )
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @Operation(
+    summary = "Search for offenders on a given staff member's caseload.",
+    description = "Search for offenders on a given staff member's caseload.. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "The query retrieved a set of results",
```

Comment:
should have a `content` section here as you are returning a payload, `licenceController.getLicenceById` has an example.

This will ensure produce swagger docs with info about the response structure